blog.md

```diff
@@ -8,69 +8,56 @@ excerpt: ""
 
-We are pleased to announce that with`gymnasium==1.0.0`, we introduce the new v5 version of the Gymnasium/MuJoCo environments. This version is highly customizable, has fixed bugs and is faster.
-`Gymnasium/MuJoCo-v5` environments and framework are available with `gymnasium==1.0.0`
+We are pleased to announce that with`gymnasium==1.0.0`, we introduce the new version 5 version of the Gymnasium/MuJoCo environments with significantly increased customizability, bugs fixes and is faster.
 ```sh
-pip install gymnasium>=1.0.0
+pip install "gymnasium[mujoco]>=1.0.0"
 ```
 
-# Key features:
-- Add support for loading third-party MuJoCo models, including realistic robot models.
--- e.g. [MuJoCo Menagerie](https://github.com/deepmind/mujoco_menagerie).
-<!--
-[MyoSim](https://github.com/facebookresearch/myosuite)
--->
+```python
+import gymnasium as gym
 
-- 80 total enhancements, notably:
--- Performance: Improves training performance by removing a considerable amount of constant 0 observations from the observation spaces of the `Ant`, `Humanoid` and `HumanoidStandup`. This results in 5-7% faster training with `SB3/PPO`-`pytorch` (related [GitHub issue](https://github.com/Farama-Foundation/Gymnasium/issues/204)).
--- Customizability: Added new arguments for all the environments and restored the removed arguments from the `v3 → v4` transition.
--- QOL: Added new fields to `info` that include all reward components and non-observable state, and `reset()` now returns `info` which includes non-observable state elements.
+env = gym.make("Humanoid-v5")
+```
 
-- A total of 24 bugs were addressed, notably:
--- In Ant, Hopper, Humanoid, InvertedDoublePendulum, InvertedPendulum, Walker2d: `healthy_reward`, was previously given on every step (even if the robot was unhealthy), now it is only given when the robot is healthy, resulting in faster learning transience. For further details about the performance impact of this change, please refer to the related [GitHub issue] (https://github.com/Farama-Foundation/Gymnasium/issues/526)
--- In Ant and Humanoid: The `contact_cost` being constantly 0, For more information on the performance impact of this change, please check the related GitHub issue.
--- In Reacher and  Pusher, the reward function was calculated based on the previous state not the current state. For further information on the performance impact of this change,  please check the related [GitHub issue](https://github.com/Farama-Foundation/Gymnasium/issues/821)).
--- Fixed several `info` fields.
--- Walker2D: Both feet now have `friction==1.9`, previously the right foot had `friction==0.9` and the left foot had `friction==1.9` (related [GitHub issue](https://github.com/Farama-Foundation/Gymnasium/issues/477)).
--- Pusher: Fixed the issue of the object being lighter than air, (related [GitHub issue](https://github.com/Farama-Foundation/Gymnasium/issues/950)).
+## Key features:
```

Comment:
Can we have a `for a complete changes made see XYZ and the related PR X`

Reply:
You mean the thing that is at the very bottom of the blog?

Reply:
Yes, but I would move it further up such that you don't need to get to the bottom to read it

Reply:
Does it really matter where it is? I believe it is preferable to be at the bottom since most users do not need to look at a complete changelog, but I can not move it if you really care